politeiawww/backend.go

```diff
@@ -24,8 +24,8 @@ import (
 	www "github.com/decred/politeia/politeiawww/api/v1"
 	"github.com/decred/politeia/politeiawww/database"
 	"github.com/decred/politeia/politeiawww/database/localdb"
+	wwwutil "github.com/decred/politeia/politeiawww/util"
 	"github.com/decred/politeia/util"
```

Comment:
Why do we have 2 utils?  Can we combine them?  If not we probably need to fold them into the regular code.

Reply:
@marcopeereboom  discussed this topic with @sndurkin. There is an util which uses a politeiawww model, and there's another util (Strip - markdown) that is general. How do you guys want to set this up?